script/Ciel/Day 3/03_01_CIEL03_3_3.txt

```diff
@@ -199,7 +199,7 @@ Seeing her smile like that sends another stab of pain through my heart.
 -- Page 172, Offset 5288.
 -- あ、シャツを脱いで体を拭いたら、そこの毛布を上着代わりにくるまっててください」
 Oh, and once you've taken your shirt off and toweled down,
- please wrap yourself in the blanket over there in lieu of a jacket."
+ please wrap yourself in the blanket over there instead of a jacket."
```

Comment:
in lieu reads to me more of a "we don't have the other one" rather than "you could do a jacket instead".